TGM/src/main/java/network/warzone/tgm/nickname/NickManager.java

```diff
@@ -18,39 +17,42 @@
 import network.warzone.tgm.modules.visibility.VisibilityController;
 import network.warzone.tgm.modules.visibility.VisibilityControllerImpl;
 import network.warzone.tgm.user.PlayerContext;
+import network.warzone.tgm.util.Strings;
 import network.warzone.warzoneapi.models.Rank;
+import network.warzone.warzoneapi.models.UserProfile;
 import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
 import org.bukkit.craftbukkit.v1_14_R1.entity.CraftPlayer;
 import org.bukkit.entity.Player;
-import org.json.JSONArray;
+import org.bukkit.event.player.PlayerTeleportEvent;
 import org.json.JSONObject;
 
 import javax.annotation.Nullable;
-import javax.net.ssl.HttpsURLConnection;
-import java.io.BufferedReader;
-import java.io.IOException;
-import java.io.InputStreamReader;
 import java.lang.reflect.Field;
-import java.net.URL;
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.Objects;
-import java.util.UUID;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.*;
 
 public class NickManager {
 
-    @Getter @Setter @AllArgsConstructor
-    public class Skin {
-        public String value;
-        public String signature;
-    }
+    public static String RATELIMITED_MESSAGE = net.md_5.bungee.api.ChatColor.GOLD.toString() + net.md_5.bungee.api.ChatColor.BOLD + "Slow Down! " + net.md_5.bungee.api.ChatColor.RESET.toString() + net.md_5.bungee.api.ChatColor.RED + "You're being ratelimited.";
```

Comment:
Any reason for using `net.md_5.bungee.api.ChatColor` when `org.bukkit.ChatColor` is already imported?